rpc/core/types/responses.go

```diff
@@ -100,6 +100,14 @@ type SyncInfo struct {
 
 	TotalSyncedTime time.Duration `json:"total_synced_time"`
 	RemainingTime   time.Duration `json:"remaining_time"`
+
+	TotalSnapshots      int64         `json:"total_snapshots"`
+	ChunkProcessAvgTime time.Duration `json:"chunk_process_avg_time"`
+	SnapshotHeight      int64         `json:"snapshot_height"`
+	SnapshotChunksCount int64         `json:"snapshot_chunks_count"`
+	SnapshotChunksTotal int64         `json:"snapshot_chunks_total"`
+	BackFilledBlocks    int64         `json:"backfilled_blocks"`
+	BackFillBlocksTotal int64         `json:"backfill_blocks_total"`
```

Comment:
We can decide which info we would like to expose to the RPC layer. Because DevOps can see these info through the metrics. I can see the info for the RPC layer might be useful when the state-sync time is long. 
For example, the current snapshot size of Crypto.com mainnet is around 250MB. In the worse case, it might take 10 mins to sync up the snapshot. For the backfill, I am not sure how many blocks need to be backfilled is reasonable. So it might take a while or it could be done in secs.
@alexanderbez @cmwaters do we want to expose these metrics to RPC?

Reply:
The time to backfill is proportional to the evidence age parameters. For 21 days thats 300,000 light blocks. I'm not sure how long that might take because I haven't done any benchmarking in that area.

The only use case I can think of is that operators might want to get a better understanding how long their state sync is going to take